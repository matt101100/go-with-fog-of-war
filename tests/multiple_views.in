view
place J10
view
place C18
view
place A1
view
place B2
view
place C6
view
place D2
view
place J11
view
place M3
view
place E5
view
place S19
view
place S18
view
place R15
view
place N12
view
term
