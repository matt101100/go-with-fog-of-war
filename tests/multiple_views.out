J10,.................................................
C18,xxxxxxxxxxxxxxx......x......x......x......x......
B18,xxxxxxxxxxxxxxxx.....xx..o..xx.....xx.....xx.....
M3,..........................................xxxxxxx
L17,xxxxxxx..........................................
B1,xx.....xx.....xx.o...xx#....xxxxxxxxxxxxxxxxxxxxx
B17,xxxxxxxxx.....xx..o..xx.....xx.....xx.....xx.....
C9,x......x......x......x......x......x......x..#...
O1,........o...................xxxxxxxxxxxxxxxxxxxxx
L12,.............................#......#............
E16,........o........................................
E18,xxxxxxxxxxxxxx........o..........................
G15,.................................................
A8,xxx....xxx....xxx....xxx....xxx....xxx..#.xxx....
